Cargo.toml

```diff
@@ -48,7 +49,7 @@ http = "0.2"
 httparse = "1"
 httpdate = "0.3.2"
 once_cell = "1"
-publicsuffix = { default-features = false, version = "1.5.4" }
+publicsuffix = "2.0.11"
```

Comment:
@algesten `publicsuffix` v2 is very light on dependencies but we can trim them even further by disabling default features if you only need to lookup unicode domains. Alternatively, we can use the `psl` crate which doesn't have any dependencies (except for `psl-types` which is already being pulled in by `addr`) but that doesn't allow the list to be updated at runtime as it's static. Finally we can ship with no PSL implementation at all which allows users to use any implementation of `psl_types::List` they like but that's a breaking change.